v19.2/table-expressions.md

```diff
@@ -85,7 +85,7 @@ For example:
 
 #### Force index selection
 
-{% include {{page.version.version}}/misc/force-index-selection.md %} 
+{% include {{page.version.version}}/misc/force-index-selection.md %}
```

Comment:
Just my $0.02 nitpick, but we should do formatting PRs separately.  No need to change in this PR, we should decide on a standard as a team.